build.psm1

```diff
@@ -31,6 +31,14 @@ if ($IsLinux) {
     $IsUbuntu14 = $IsUbuntu -and $LinuxInfo.VERSION_ID -match '14.04'
     $IsUbuntu16 = $IsUbuntu -and $LinuxInfo.VERSION_ID -match '16.04'
     $IsCentOS = $LinuxInfo.ID -match 'centos' -and $LinuxInfo.VERSION_ID -match '7'
+    $IsFedora = $LinuxInfo.ID -match 'fedora' -and $LinuxInfo.VERSION_ID -ge 24
+    $IsRedHatFamily = $IsCentOS -or $IsFedora
+
+    # Workaround for temporary LD_LIBRARY_PATH hack for Fedora 24
+    if (Test-Path ENV:\LD_LIBRARY_PATH) {
```

Comment:
The comment implies that this applies only to Fedora 24. Should we add an additional condition to the IF statement?
`
if ($IsFedora -AND Test-Path ENV:LD_LIBRARY_PATH) {
`